### DISCUSSION TITLE
Project 3

### DISCUSSION THREAD

**Discussion by cgallagher1021** [cooperrc/computational-mechanics#164] [Q&A] [answered]
Can anyone tell me where my syntax error in line 3 is? I feel like it is something simple and right in front of my face but I'm drawing a blank. 
<img width="504" alt="Capture" src="https://user-images.githubusercontent.com/121522937/211442489-5b192b5f-803f-403a-9842-cc3566f52167.PNG">

**Comment by DJA17** [comment, marked as answer]
It may be that you haven't imported the correct libraries to plot (import matplotlib.pyplot as plt). If that doesnt work maybe try looking at the label; I'm not sure if it needs to be with in brackets { } or ' '.

**Reply by C-Doll** [reply]
sol.y[0][-1]
Should it be
sol.y[:-1] 
or something along those lines? I haven't looked at the third project yet.

**Reply by DJA17** [reply]
It could be that. It may also be that you havent defined t in the for loop (t = np.linspace(0,(m0-mf)/dmdt,20). Also Im wondering what the full line of sol = solve_ivp is since there may be in error in that line that only flags when you call it in line 3.

**Reply by cgallagher1021** [reply]
This was the issue, in line 2 I had a typo when meaning to type np.linspace and the following information in the parenthesis. This caused the invalid syntax in line three. I was able to fix it. Thank you both for your help!
